src/Game.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+function Game(){
+  this._rollNumber = 1;
+  this._frameNumber = 1;
+  this._score=0;
+}
+
+
+Game.prototype.frameNumber = function(){
+  return this._frameNumber;
+};
+
+Game.prototype.rollNumber = function(){
+  return this._rollNumber;
+};
+
+Game.prototype.score = function(){
+  return this._score;
+};
+
+Game.prototype.increaseScore = function(roll){
```

Comment:
This function has too many statements. (12)<br>This function's cyclomatic complexity is too high. (5)